Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Golang build container
+FROM golang:1.10
+
+WORKDIR $GOPATH/src/github.com/grafana/grafana
+
+COPY Gopkg.toml Gopkg.lock ./
+COPY vendor vendor
```

Comment:
On a fresh git clone, this will fail as `/vendor` is listed in `.dockerignore`

Reply:
Removed /vendor from .dockerignore